### DISCUSSION TITLE
How do I change the color for "matching" chars? (0.11.2-git)

### DISCUSSION THREAD

**Discussion by paru-de** [abenz1267/walker#194] [Q&A] [answered]
Hey,

I've been using Walker for a while now with the default Catppuccin theme slightly adjusted to my liking. However, a few days ago, I noticed that all my searches now have a turquoise highlighting color for matching chars:

![image](https://github.com/user-attachments/assets/4af3dab3-8acb-4b18-9ca3-dc7ddf07dd75)

Since I'm using the flake I assume that you added this as a new feature a few days ago (probably 64e8072 / f691c52?)  and I would like to style it but I don't know which element I need to access (I had a look through [the wiki](https://github.com/abenz1267/walker/wiki/Theming) but must have missed it).

Would appreciate help,
Thanks and happy holidays!

Patrick

**Comment by abenz1267** [comment, marked as answer]
Use `ui.window.box.scroll.list.marker_color` in your theme's layout file.

**Reply by paru-de** [reply]
Awesome, thank you for the quick reply!
